### ISSUE TITLE
Option to include  Xrm.WebApi to D365-Ui-Test

### ISSUE THREAD

**Issue by victorsolaya** [XRM-OSS/D365-UI-Test#17]
Hi @DigitalFlow 

I am finding a case that I would need to retrieve values from the WebApi.
Is there any possibility to use it at the moment or do you have any plan to include it?

Thank you!

**Comment by DigitalFlow** [comment]
Hi @victorsolaya,

I can add it, but you can also use this small function in the meantime:

```TypeScript
        await EnsureXrmGetter(this._page);

        await this._page.evaluate(([entity, id]) => {
            const xrm = window.oss_FindXrm();

            return xrm.WebApi.retrieveRecord(entity, id);
        }, [ "account", "49fd549f-a589-4bef-84d2-b6de1208bbb9" ]);
```

EnsureXrmGetter is from here: https://github.com/XRM-OSS/D365-UI-Test/blob/master/src/xrm/Global.ts

Hope this helps, I'll try adding the most commonly functions of WebAPI in the next release.

Kind Regards,
Florian

**Comment by DigitalFlow** [comment]
Hi @victorsolaya,
`XrmUITest.WebApi` is now included in v0.1.22, please give it a try :)

Kind regards,
Florian

**Comment by victorsolaya** [comment]
Love it! Man thanks @DigitalFlow . Brilliant work as always
